### PR TITLE
Add AnalysisConnector::FILE_TRANSFER

### PR DESCRIPTION
**Pull request by sseckler**
Add AnalysisConnector::FILE_TRANSFER to add support for file transfers.

### CODE REVIEW

**Comment by sseckler** [comment]
@rogerta @dominiquefc can one of you review this PR?

File: proto/content_analysis/sdk/analysis.proto [hunk 1]
```diff
@@ -37,6 +37,7 @@ enum AnalysisConnector {
   FILE_ATTACHED = 2;
   BULK_DATA_ENTRY = 3;
   PRINT = 4;
+  FILE_TRANSFER = 5;
```
Comment (rogerta) [inline proto/content_analysis/sdk/analysis.proto:40]: Please leave a comment above this value explaining that this connector is not yet implemented in the SDK but the value is present here to remain consistent with the equivalent chromium code.

**Review by rogerta — CHANGES_REQUESTED** [review]
Thanks Steffen!  Please see below for one comment and then I'll merge the PR.

File: proto/content_analysis/sdk/analysis.proto (hunk 1 above)
Comment (sseckler) [inline proto/content_analysis/sdk/analysis.proto:40]:
```suggestion
  // This value is not yet implemented in the SDK. It is kept for consistency with the Chromium code.
  FILE_TRANSFER = 5;
```

**Pull request merged**